static/js/portico/signup.js

```diff
@@ -4,7 +4,13 @@ $(function () {
 
     $.validator.addMethod('password_strength', function (value) {
         return common.password_quality(value, undefined, $('#id_password, #id_new_password1'));
-    }, "Password is too weak.");
+    }, function () {
+        var password_input = $('#id_password, #id_new_password1');
+        if (password_input.is(':focus')) {
```

Comment:
I made some changes. So the current model is like this.
*  Start showing the error messages once the user defocuses for the first time.
* When the user starts typing again after the defocus the error will stay or may change to a different error depending upon what he types. The error can also go off completely and the box may turn green as he types.
Is this what you have in mind?